app/src/lib/stats/stats-database.ts

```diff
@@ -134,9 +134,15 @@ export interface IDailyMeasures {
    */
   readonly enterpriseCommits: number
 
-  /** The number of time the user made a commit to a repo hosted on Github.com */
+  /** The number of times the user made a commit to a repo hosted on Github.com */
   readonly dotcomCommits: number
 
+  /** The number of times the user made a commit to a protected GitHub or GitHub Enterprise repository */
+  readonly commitsToProtectedBranch: number
+
+  /** A flag indicating the user made a commit to a repository with branch protections enabled */
+  readonly commitWithBranchProtectionsEnabled: boolean
```

Comment:
non-blocking, but perhaps `committedInRepoWithBranchProtections` could be more clear? to make it more clear that the repo has the branch protections and not that a commit was made on a protected branch.

(the doc for this is totally clear to me though)

Reply:
Gonna hold of on this potential rename until we can confirm we're not misunderstanding the requirements in https://github.com/desktop/desktop/pull/7595#issuecomment-494925379

Reply:
After further discussions this has been renamed to `commitsToRepositoryWithBranchProtections` in https://github.com/desktop/desktop/pull/7595/commits/6520ce892c86cd65434341e5c5bb056284b219be